ReflectometryServer/test_modules/__init__.py

```diff
@@ -11,7 +11,7 @@ def load_tests(loader, standard_tests, pattern):
     The tests in this module are only added under Python 2.
```

Comment:
update comment

Reply:
ok, feel free to do this for me in the future but I will do this now :-)